### ISSUE TITLE
RHEL information is incorrect/incomplete

### ISSUE THREAD

**Issue by glandium** [glandium/firefox-linux-compat-matrix#7]
- EOL dates for intermediate versions (e.g. 7.x < 7.9, 8.x < 8.6) are obviously wrong.
- I think there are extra versions of clang/GCC available

@stransky could you take a look into those? Relatedly, do RHEL7 and 8 really only have python 3.6?

**Comment by stransky** [comment]
Hi Mike, I'll ask for the dates but they look almost correct.

The clang/GCC versions available it difficult to say. We regularly rebase gcc/clang/rust to build latest ESR Firefox line. So we first check what's needed by Firefox and then do the update and the updated gcc/clang/rust are available in additional channel so it may not be available for all customers. It's kind of chicken/egg problem here.

**Comment by stransky** [comment]
Yes, base rhel7/8 has python 3.6 but there are additional channels with 3.9
